components/ExclusiveContentFromTheLab.vue

```diff
@@ -0,0 +1,54 @@
+<script setup>
+import { onMounted, ref } from 'vue'
+import axios from 'axios'
+import Skeleton from 'primevue/skeleton'
+const dataLoaded = ref(false)
+const episodes = ref([])
+onMounted(async () => {
```

Comment:
look into the earliest possible lifecycle hook for these: https://vuejs.org/api/composition-api-lifecycle.html